exercises/allergies/test/test_allergies.c

```diff
@@ -10,34 +10,326 @@ void tearDown(void)
 {
 }
 
-static void test_no_allergies_means_not_allergic(void)
+static void test_not_allergic_to_anything_for_eggs(void)
 {
    unsigned int score = 0;
 
-   TEST_ASSERT_FALSE(is_allergic_to(ALLERGEN_PEANUTS, score));
-   TEST_ASSERT_FALSE(is_allergic_to(ALLERGEN_CATS, score));
-   TEST_ASSERT_FALSE(is_allergic_to(ALLERGEN_STRAWBERRIES, score));
+   TEST_ASSERT_FALSE(is_allergic_to(ALLERGEN_EGGS, score));
 }
 
-static void test_is_allergic_to_eggs(void)
+static void test_allergic_only_to_eggs(void)
 {
    TEST_IGNORE();               // delete this line to run test
    unsigned int score = 1;
 
    TEST_ASSERT_TRUE(is_allergic_to(ALLERGEN_EGGS, score));
 }
 
-static void test_is_allergic_to_eggs_in_addition_to_other_stuff(void)
+static void test_allergic_to_eggs_and_something_else(void)
 {
    TEST_IGNORE();
-   unsigned int score = 5;
+   unsigned int score = 3;
+
+   TEST_ASSERT_TRUE(is_allergic_to(ALLERGEN_EGGS, score));
+}
+
+static void test_allergic_to_something_but_not_eggs(void)
+{
+   TEST_IGNORE();
+   unsigned int score = 2;
+
+   TEST_ASSERT_FALSE(is_allergic_to(ALLERGEN_EGGS, score));
```

Comment:
I know you're just copying the style of the existing tests, but how about this:

```suggestion
   TEST_ASSERT_FALSE(is_allergic_to(ALLERGEN_EGGS, 2));
```

I'm not sure how introducing a local helps to do anything except create separation between the input and the expected output.

Reply:
Done

Reply:
Thanks!